nestor_api/lib/kubectl.py

```diff
@@ -0,0 +1,44 @@
+"""Kubernetes' kubectl library"""
+
+from enum import Enum
+import json
+import os
+
+from nestor_api.config.kubernetes import KubernetesConfiguration
+import nestor_api.lib.io as io
+
+
+class KubernetesResourceType(Enum):
+    """An enum representing the different resources kinds"""
+
+    DEPLOYMENT = "deployment"
+
+    def __str__(self):
+        return str(self.value)
```

Comment:
Maybe we could define this under `k8s` in a dedicated file.
Also, the file `kubectl` could be `cli.py` under `k8s`, what do you think, it is just to avoid spreading all k8s logic in the project.